generic.go

```diff
@@ -171,3 +243,11 @@ type Holder struct {
 	objectProcessCount prometheus.Counter
 	objectProcessTime  prometheus.Histogram
 }
+
+type ServerHolder struct {
+	Server       Server
+	ZapNameField ZapNameField
+	ReadyForWork <-chan struct{}
```

Comment:
This is unused

Reply:
Too later. I've created https://github.com/atlassian/ctrl/issues/6